### PR TITLE
Issue 11: hdfs - remove common

### LINKED ISSUE #11: migrate hdfs
##Problem description
The hdfs binding in the Pravega main source needs to be moved to this repository. Few criteria pertaining to the changes for this purpose:
1) they should be specific to hdfs in pravega main source, little or no additional files if possible
2) they should preserve history across repositories
3) they should include all the types of tests in this repository.
4) they should be as much as in sync with the Pravega main source as possible
5) they should not require to be imported elsewhere
6) they should handle all workflows using this binding

##Problem location
replace hdfs files in the new repo from those in the Pravega main source

##Suggestions for an improvement
- separation of pravega version for src,test,docker even though they may point to the same version

### PR DESCRIPTION
**Pull request by ravibeta**
Change log description
This change is for removal of common now the sourceset of common is available from master.

Fixes #11

Purpose of the change
Fixes #11

What the code does
It does cleanup as suggested to be waited for until the changes are available in master.

How to verify it
This builds and tests as before. No change to java/src/main or to functionality.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -104,16 +120,18 @@ allprojects {
             //failOnVersionConflict()
             force "com.google.guava:guava:" + guavaVersion
             force "org.slf4j:slf4j-api:" + slf4jApiVersion
+            force "com.google.code.gson:gson:2.5"
```
Comment (sachin-j-joshi) [inline build.gradle]: why is this needed?

Comment (ravibeta) [reply build.gradle]: Yes Sachin

**Review by sachin-j-joshi — COMMENTED** [review]
LGTM.

Note - that having libs is only a temporary solution , and need to be removed once publishing related changes are merged in pravega repository

**Review by sachin-j-joshi — APPROVED** [review]
LGTM.
